frontend/src/components/ShootDns/DnsConfiguration.vue

```diff
@@ -11,11 +11,11 @@ SPDX-License-Identifier: Apache-2.0
     ref="actionDialog"
     caption="Configure DNS"
     width="1000"
-    confirmRequired
+    confirm-required
     @dialog-opened="onConfigurationDialogOpened"
     >
     <template v-slot:actionComponent>
-      <manage-shoot-dns :key="componentKey"></manage-shoot-dns>
+      <manage-shoot-dns :key="componentKey" :noCustomDomain="!isCustomShootDomain"></manage-shoot-dns>
```

Comment:
I absolutely do not understand why we need to introduce this component parameter. I though the goal of my last my last PR was to get rid of these kind of component parameters. All the necessary information is already available on the store.